src/coreclr/inc/corinfo.h

```diff
@@ -1720,6 +1720,7 @@ struct CORINFO_FIELD_INFO
     CORINFO_HELPER_DESC     accessCalloutHelper;
 
     CORINFO_CONST_LOOKUP    fieldLookup;        // Used by Ready-to-Run
+    void*                   knownFieldAddress;
```

Comment:
It would be better to just use the existing `fieldLookup` field for this instead of adding new one.

Reply:
(Or even always return the static field address in `fieldLookup`.)